### PR TITLE
create all the routes for the project

### PR DESCRIPTION
**Pull request by lady-rosie**
I create the routes, can you look at it please :)

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,4 +2,48 @@
...
+  end
+
+  # MVP- Update and destroy plots
+  resources 'plots', only: [:update, :edit, :destroy] do
+    #MVP - create plants inside a plot
+    resources 'plants', only: [:create]
+  end
+  #MVP - Watering see the all the watering, one specific watering and update
+  resources 'waterings', only: [:index, :show, :update]
+
```
Comment (jscouller09) [inline config/routes.rb]: make custom path: patch waterings/:id/complete, to: waterings#mark_as_complete

File: config/routes.rb [hunk 2]
```diff
@@ -2,4 +2,48 @@
...
+  # MVP - Destroy plants
+  resources 'plants', only: [:destroy]
+
+  #MVP - Tasks index | IMPORTANT create update and destroy (the creation)
+  resources 'tasks', only: [:index, :create, :update, :destroy]
+  #MVP - Mark task as complete
+  patch 'tasks/:id', to: 'tasks#mark_as_complete'
```
Comment (jscouller09) [inline config/routes.rb]: change path to: tasks/:id/complete

File: config/routes.rb [hunk 3]
```diff
@@ -2,4 +2,48 @@
...
+
+  #IMPORTANT - mark a plot as watered
+  patch 'plots/:id', to: 'plots#complete_watering'
```
Comment (jscouller09) [inline config/routes.rb]: change path to: plots/:id/complete_waterings

File: config/routes.rb [hunk 4]
```diff
@@ -2,4 +2,48 @@
...
+
+  #IMPORTANT - See current weather, forecast and alerts
+  resources 'weather_stations', only: [:show]
+
+  #NICE TO HAVE - Display all conversations ans create a message
```
Comment (jscouller09) [inline config/routes.rb]: typo: and

File: config/routes.rb (hunk 2 above)
Comment (lady-rosie) [reply config/routes.rb]: done

File: config/routes.rb (hunk 1 above)
Comment (lady-rosie) [reply config/routes.rb]: done

File: config/routes.rb (hunk 3 above)
Comment (lady-rosie) [reply config/routes.rb]: done

File: config/routes.rb (hunk 4 above)
Comment (lady-rosie) [reply config/routes.rb]: done

**Pull request merged**